source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,243 @@
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin caseSensitiveExample
+   :end-before: end caseSensitiveExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 2, "name": "Discrete Mathematical Structures for Computer Science", "major": ["Computer Science", "Mathematics"] }
+   { "_id": 3, "name": "Environmental Science", "major": ["Environmental Science", "Sustainable Management"] }
+
+.. note::
+
+   The output would consist of all the documents in the
+   ``junior_courses`` collection if the search was not case sensitive
+   because each document contains the term "Science" in the ``name``
+   field. 
+
+Search Text By a Phrase
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass the phrase with escaped quotes to the ``Filters.text()`` method to
+search for a phrase in your text search. If you don't add escaped quotes,
+the method performs a :ref:`term search <term_search>`. 
+
+Example 
+```````
+
+The student wants to find Computer Science courses.
+
+To address this scenario, the student performs a text search on
+documents that contain the phrase "Computer Science":
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin phraseExample
+   :end-before: end phraseExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 2, "name": "Discrete Mathematical Structures for Computer Science", "major": ["Computer Science", "Mathematics"], "score": 1.2 }
+   { "_id": 4, "name": "Image Processing in Computer science", "major": ["Artificial Intelligence", "Computer Science"], "score": 1.25 }
+ 
+Search Text With Terms Excluded
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Prefix a minus sign ``-`` to each term you want to exclude in your text
+search. 
+
+Example 
+```````
+The student wants to only find Science courses.
+
+To address this scenario, the student performs a text search on
+documents that contain the term "Science", but do not contain the term
+"Computer": 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin negateExample
+   :end-before: end negateExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "name": "Forensic science", "major": ["Forensic Science", "Criminal Justice"] }
+   { "_id": 3, "name": "Environmental Science", "major": ["Environmental Science", "Sustainable Management"] }
+  
+Sorting by Relevance
+~~~~~~~~~~~~~~~~~~~~
+
+You can retrieve a score to see how relevant the returned documents are
+in your text search. To retrieve a score, project the ``metaTextScore``
+in your query.
+
+Example 
+```````
+
+The student wants to find Computer Science courses and view the courses
```

Comment:
You can use the pronoun "them" to refer to "the courses" since they're the last object prior to that in the sentence.